Basic/ClientDriven/Assets/Scripts/ClientPlayerMove.cs

```diff
@@ -1,6 +1,9 @@
+using System;
```

Comment:
Adding a note here, not linked to this file.
wanna add an entry to the changelog please?